deploy/charts/arcadia/values.yaml

```diff
@@ -41,13 +63,32 @@ fastchat:
 # @param oss.enabled Enable Object-Storage-Service in arcadia
 # @param oss.buckets List of default buckets in arcadia
 minio:
+  image:
+    repository: kubeagi/minio
+    tag: RELEASE.2023-02-10T18-48-39Z
   mode: standalone
   rootUser: "admin"
   rootPassword: "Passw0rd!"
   persistence:
     enabled: true
-    storageClass: ""
+    storageClass: "openebs-hostpath"
```

Comment:
Can we keep this empty by default?  Most probably user shall use the default storageclass.